CONTRIBUTING.md

```diff
@@ -15,10 +15,28 @@ echo '#!/bin/sh' > .git/hooks/pre-commit && echo 'composer pre-commit' >> .git/h
 - Run `composer phpcs` for the changed PHP files before submitting your PR for review (optional). 
 - Keep PRs focused and change the minimum number of lines to achieve your goal.
 
-To run integration tests on the SDK, you'll need to create a `.env` file in the root of this package with the following entries:
+### Running unit tests
+
+Prior to making a PR, ensure that the unit tests pass:
+
+`composer test-unit`
+
+### Running integration tests
```

Comment:
We may want to consider simplifying the required setup to run integration tests at some point. But first step is to document how to run currently.